src/tests/functional/plugin/shared/include/behavior/ov_plugin/auto_batching_tests.hpp

```diff
@@ -86,7 +86,7 @@ class AutoBatching_Test : public OVPluginTestBase,
             for (size_t j = 0; j < num_requests; j++) {
                 outputs.push_back(output);
                 outElementsCount.push_back(
-                        std::accumulate(begin(fn_ptrs[i]->get_output_shape(0)), end(fn_ptrs[i]->get_output_shape(0)), 1,
+                        std::accumulate(begin(fn_ptrs[i]->get_output_shape(0)), end(fn_ptrs[i]->get_output_shape(0)), size_t(1),
```

Comment:
please, fix a compilation error introduced in current PR
See https://github.com/openvinotoolkit/openvino/actions/runs/7672375947/job/20912751386?pr=22469#step:15:6782